docs/concepts/demisto-sdk.md

```diff
@@ -282,13 +282,19 @@ You have a responsibility to identify and eliminate any secrets before they get
 #### Ignoring secrets
 
 At times there is information that might be flagged incorrectly by the command as a secret. Or it may be "secret" but we wish to share it publicly (for example the support email address).
-To ignore a specific secret, enter it to the packs's `.secrets-ignore` file.
+
+- To ignore a specific secret, add it into the packs's `.secrets-ignore` file.
 For example:
 
-```text
-https://myurl.com
-https://my2ndUrl.com
-```
+    ```bash
+    echo https://myurl.com >> ./Packs/$PACK_NAME/.secrets-ignore
+    ```
+
+- To ignore a complete file, push the file path (relative to the `content` repo root) to the `.files` array in `./Tests/secrets_white_list.json`:
+
```

Comment:
I am not sure we want this documented externally I fear it might be abused. WDYT?

Reply:
Maybe better to put it only in the SDK docs.

Reply:
Yea, I agree, better to put this in the sdk instead.